test/auth/credential-provider.test.ts

```diff
@@ -4,9 +4,9 @@ import {
 } from '../../src/auth/credential-provider';
 
 const testToken =
-  'eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJmb29AYmFyLmNvbSIsImNwIjoiY29udHJvbC1wbGFuZS1lbmRwb2ludC5iYXIuY29tIiwiYyI6ImNhY2hlLWVuZHBvaW50LmJhci5jb20ifQo.rtxfu4miBHQ1uptWJ2x3UiAwwJYcMeYIkkpXxUno_wIavg4h6YJStcbxk32NDBbmJkJS7mUw6MsvJNWaxfdPOw';
-const testControlEndpoint = 'control-plane-endpoint.bar.com';
-const testCacheEndpoint = 'cache-endpoint.bar.com';
+  'eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJmb29Abm90LmEuZG9tYWluIiwiY3AiOiJjb250cm9sLXBsYW5lLWVuZHBvaW50Lm5vdC5hLmRvbWFpbiIsImMiOiJjYWNoZS1lbmRwb2ludC5ub3QuYS5kb21haW4ifQo.rtxfu4miBHQ1uptWJ2x3UiAwwJYcMeYIkkpXxUno_wIavg4h6YJStcbxk32NDBbmJkJS7mUw6MsvJNWaxfdPOw';
+const testControlEndpoint = 'control-plane-endpoint.not.a.domain';
+const testCacheEndpoint = 'cache-endpoint.not.a.domain';
```

Comment:
domain is not a TLD, but domains is. 😁 

`example.*` and `test.*` are safe.
https://en.wikipedia.org/wiki/Special-use_domain_name

Not a blocker.

Reply:
TIL.  Are those actually enforced somehow / somewhere?  I would bet money that I could go create a route53 entry that contained a path segment `example.` or `test.`!  Would be really surprised if they blocked those but maybe?!

Reply:
```
ping example.com
PING example.com (93.184.216.34): 56 data bytes
64 bytes from 93.184.216.34: icmp_seq=0 ttl=56 time=7.332 ms
64 bytes from 93.184.216.34: icmp_seq=1 ttl=56 time=10.797 ms
64 bytes from 93.184.216.34: icmp_seq=2 ttl=56 time=11.262 ms
64 bytes from 93.184.216.34: icmp_seq=3 ttl=56 time=7.934 ms
64 bytes from 93.184.216.34: icmp_seq=4 ttl=56 time=11.182 ms
64 bytes from 93.184.216.34: icmp_seq=5 ttl=56 time=7.005 ms
```

Reply:
> Example Domain
>
> This domain is for use in illustrative examples in documents. You may use this domain in literature without prior coordination or asking for permission.